astropy/cosmology/core.py

```diff
@@ -99,13 +99,13 @@ class FLRW(Cosmology):
         Neff. Usually this means you must provide three neutrino masses unless
         you are considering something like a sterile neutrino.
 
-    name : str
-        Optional name for this cosmological object.
-
-    Ob0 : float
+    Ob0 : float or None
```

Comment:
I believe it is encouraged in the contributing docs somewhere, though it's one of those small details that hasn't really been enforced consistently, so it's no big deal.  Thanks for adding it though!